plugins/web/opentelemetry-plugin-document-load/src/index.ts

```diff
@@ -15,3 +15,4 @@
  */
 
 export * from './documentLoad';
```

Comment:
I totally agree, but I think we should wait for the plugin design to be finalized on this [issue](https://github.com/open-telemetry/opentelemetry-js/issues/1207) since the tracer and meter plugin may end up being the same.